build/elasticsearch/bin/es-docker

```diff
@@ -41,7 +41,7 @@ do
     then
         if [[ ! -z $envvar_value ]]; then
           es_opt="-E${envvar_key}=${envvar_value}"
-          es_opts+=" ${es_opt}"
+          es_opts+=("${es_opt}")
```

Comment:
The same thing needs to be done to line 51 further down (there are now two places where we append to the array).